packages/calcite-components/src/components/input/common/tests.ts

```diff
@@ -32,8 +32,8 @@ export function testPostValidationFocusing(
     const hiddenInputSelector = `input[slot=${hiddenFormInputSlotName}]`;
     const inputSelector = `${inputTag}[name=${inputName}]`;
 
-    expect(await isElementFocused(page, hiddenInputSelector)).toBe(true);
-    expect(await isElementFocused(page, inputSelector)).toBe(false);
+    expect(await isElementFocused(page, hiddenInputSelector)).toBe(false);
+    expect(await isElementFocused(page, inputSelector)).toBe(true);
```

Comment:
> do you remember if there was a reason that the hidden input needs to be focused after form submission?

Yes, it's because the browser will focus and display the validation message on the hidden input. This is no longer the case with your changes, right? 🚀

Reply:
yup!